client/my-sites/stats/activity-log/utils.js

```diff
@@ -1,5 +1,10 @@
 /** @format */
 /**
+ * External dependencies
+ */
+import momentLib from 'moment-timezone';
```

Comment:
Interesting to note that we are now using a non-localised moment lib, instead of the one supplied by `localize()`.  Given the way we use it, that shouldn't make any difference.